core/shared/src/main/scala/fs2/async/Ref.scala

```diff
@@ -15,7 +15,7 @@ import java.util.concurrent.atomic.{AtomicBoolean,AtomicReference}
 import Ref._
 
 /** An asynchronous, concurrent mutable reference. */
-final class Ref[F[_],A](implicit F: Effect[F], ec: ExecutionContext) { self =>
+final class Ref[F[_],A] private[fs2] (implicit F: Effect[F], ec: ExecutionContext) { self =>
```

Comment:
We could make this `private` instead and change async.scala to use `Ref.uninitialized`

Reply:
That was my initial thought, then worried about the extra method call. Arguably over optimising though. (could also do `private[async]`).
Should I change it as you suggest?

Reply:
scratch it, `private[async]` won't work since it's used in Scope as well

Reply:
Ah right, merging then